codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -276,22 +277,25 @@ public class <%= schema_name %> {
                 <% end %>
 
                 <% type.required_input_fields.each do |field| %>
+                  <%= java_annotations(field) -%>
                   public <%= java_input_type(field.type) %> get<%= field.classify_name %>() {
                       return <%= escape_reserved_word(field.camelize_name) %>;
                   }
 
-                  public <%= type.name %> set<%= field.classify_name %>(<%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
+                  public <%= type.name %> set<%= field.classify_name %>(<%= java_annotations(field).gsub(/(.)$/, '\1 ') %><%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>) {
```

Comment:
Yes, I think the correct solution is to let `java_annotations` format differently for arguments vs methods or fields. I'm gonna add some tests for `java_annotations` while I'm at it